ci-operator/jobs/devfile/registry/devfile-registry-main-periodics.yaml

```diff
@@ -10,20 +10,18 @@ periodics:
     org: devfile
     repo: registry
   labels:
-    ci-operator.openshift.io/variant: v4.12
+    ci-operator.openshift.io/variant: v4.13
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-devfile-registry-main-v4.12-rhtap-nightly-devfile-compatibility
+  name: periodic-ci-devfile-registry-main-v4.13-rhtap-nightly-devfile-compatibility
   reporter_config:
     slack:
-      channel: '#team-has-github-alerts'
+      channel: '#ops-ci'
       job_states_to_report:
-      - success
       - failure
       - error
-      report_template: '{{ if eq .Status.State "success" }} :success-kid: {{ else
-        }} :fuuuuuu: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs>'
+      report_template: '@aws-ci-admin Job *{{.Spec.Job}}* failed. <{{.Status.URL}}|View
+        logs>'
```

Comment:
Might of been a mix up with the reverted changes.

Reply:
Most probably. I've reverted back the original slack notification version.